lib/utils.pm

```diff
@@ -1115,7 +1115,7 @@ sub get_x11_console_tty {
     # older versions in HDD
     my $newer_gdm
       = $new_gdm
-      && !is_sle('<=15-SP2')
+      && !is_sle('<15-SP2')
```

Comment:
Maybe I am missing something obvius, but, why are the conditions so obscure?
Isn't this simpler:
```suggestion
      && is_sle('>=15-SP2')
```

Reply:
Well, because they are not the same. Just take the example of "Tumbleweed" and check against the `is_sle` calls:

* is_sle('<15-SP2') -> false and !is_sle('<15-SP2') -> true
* is_sle('>=15-SP2') -> false

Reply:
I still think that is_sle('>=15-SP2') and is_leap('>15.1') should both return true for tumbleweed.
But, another topic for another day.